packages/cubejs-query-orchestrator/src/driver/BaseDriver.js

```diff
@@ -199,8 +199,7 @@ export class BaseDriver {
     const rows = await this.query(query, values);
     if (rows.length === 0) {
       throw new Error(
-        'Unable to detect column types for pre-aggregation on empty values in readOnly mode. \n' +
```

Comment:
We need a link for this with a description about limitation of types detection via heuristics (doesn't work on empty response)

Reply:
@ovr @hassankhan what database affected here? I assume Postgres and MySQL don't have this issue, because they are read-only by default.

Reply:
I'm sorry for the late reply.

> I assume Postgres and MySQL don't have this issue,

Yes, it's right!.

> what database affected here?

Druid, DreamIO, Athena, Hive, BigQuery and etc. For some of them, we can prepare a PR to fetch types for columns from the driver/database, but for example, Druid doesn't return metadata for the result's set.